src/components/templates/AnimateMap/AnimateMap.tsx

```diff
@@ -61,47 +65,45 @@ export const AnimateMap: React.FC<AnimateMapProps> = ({ space }) => {
       containerRef.current as HTMLDivElement
     );
 
-    await game.init();
-    await game.start();
-
     setApp(game);
-  }, [containerRef, app, dataProvider, store, space]);
-
-  useEffect(() => void initialize(), [initialize]);
-  useEffect(() => () => void app?.release(), [app]);
+    //eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Please add the appropriate deps or a comment in the code explaining why it is necessary to ignore dependencies and what that may cost.

```suggestion
```